pkgs/games/vassal/default.nix

```diff
@@ -1,19 +1,21 @@
-{ stdenv, fetchurl, jre, makeWrapper }:
+{ stdenv, fetchzip, jre, makeWrapper }:
 
 stdenv.mkDerivation rec {
-  name = "VASSAL-3.3.2";
+  pname = "VASSAL";
+  version = "3.4.12";
+  name = "${pname}-${version}";
```

Comment:
```suggestion
```

Reply:
Doesn't the derivation need a `name` argument?

Reply:
According to this: http://blog.ielliott.io/nix-docs/stdenv-mkDerivation.html

name is set to "${pname}-${version}"

Reply:
I didn't know that. Updated